### ISSUE TITLE
Some unrecognized characters

### ISSUE THREAD

**Issue by arhowe00** [joouha/euporie#91]
Most windows display the following
![image](https://github.com/joouha/euporie/assets/38041948/68f119bf-fac7-45f6-8616-22c95c21ddbe)

**Comment by joouha** [comment]
Hi,

You'll probably need to install something like [awesome-terminal-fonts](https://github.com/gabrielelana/awesome-terminal-fonts) or [nerdfonts](https://www.nerdfonts.com/) to get the extra glyphs to display these characters.

Unfortunately there is no simple / universal way to detect which characters a terminal is able to display. There are many scenarios where this would not be possbile - e.g. euporie could be running in an SSH session and displayed in a terminal on a completely separate computer, where the is no way to check which fonts are installed.

**Comment by arhowe00** [comment]
Yeah, that's what I assumed.

Are you sure it's best to enable this by default?  It might make this less portable since I assume CLI tools are more portable

**Comment by joouha** [comment]
Agreed, I'll turn them off by default (This will be a setting in the next release). 

The only tricky character remaining in your screenshot is `🮇` (right one quarter block), which I use for box drawing in various places. It's from the [Symbols for Legacy Computing](https://en.wikipedia.org/wiki/Symbols_for_Legacy_Computing) unicode block which was part of Unicode 13.0. I'm surprised that it's not available in the default fonts on modern operating systems. Which OS are you using?

**Comment by MichalNemecek** [comment]
> You'll probably need to install something like [awesome-terminal-fonts](https://github.com/gabrielelana/awesome-terminal-fonts) or [nerdfonts](https://www.nerdfonts.com/) to get the extra glyphs to display these characters.

It should be noted that not all Nerd Fonts work. It could be a problem with Windows Terminal, but for me Iosevka Nerd Font works fine:
![image](https://github.com/joouha/euporie/assets/37672247/cfccef40-2226-4f65-8b42-4b0b52d2a7e1)

 but FiraCode, which I normally use, and BigBlueTerm don't:
![image](https://github.com/joouha/euporie/assets/37672247/0448df59-4a62-4cf7-9ba2-82942966b92e)
![image](https://github.com/joouha/euporie/assets/37672247/9b236bb5-83c9-4c9d-9dea-e9cca573d6e2)
